docs/internals/contributing/writing-code/unit-tests.txt

```diff
@@ -121,12 +121,22 @@ Going beyond that, you can specify an individual test method like this::
 Running the Selenium tests
 --------------------------
 
-Some tests require Selenium and a Web browser (Firefox, Google Chrome, or
+Some tests require Selenium(>=2.45) and a Web browser (Firefox, Google Chrome, or
 Internet Explorer). To allow those tests to be run rather than skipped, you must
 install the selenium_ package into your Python path and run the tests with the
-``--selenium`` option::
+``--selenium=<BROWSER>`` option.
+For example, using Firefox (``--selenium=firefox``)::
 
-   $ ./runtests.py --settings=test_sqlite --selenium admin_inlines
+    $ ./runtests.py --selenium=firefox admin_views.tests.SeleniumTests
+
+Depending on which browsers are installed on your system, you may use the
+following values for the ``--selenium`` option:
+
+* ``firefox``: Firefox
+* ``opera``: Opera
+* ``ie``: Internet Explorer
+* ``chrome``: Google Chrome
+* ``phantomjs``: PhantomJS
```

Comment:
I suggested that `runtests.py --selenium` (with no spec) should raise an error with the possible choices. Then we can document that as the way to get possible values and don't have to keep the doc up to date as browsers come and go.